node/lib/DeviceAccessor.ts

```diff
@@ -88,7 +177,7 @@ export class DeviceAccessor {
                 setPropertyMethod = deviceInterface[methodName + "Async"];
```

Comment:
It is created as `"set" + capitalize(propertyName)` 3 lines above.

---
In reply to: [73293530](https://github.com/opent2t/opent2t/pull/2#discussion_r73293530) [](ancestors = 73293530)